Pod/Classes/TEDomain.m

```diff
@@ -94,6 +94,16 @@ - (void)sendActionToMiddlewares:(TEBaseAction *)action
     }];
 }
 
+- (void)registerTemporaryStore:(TEBaseStore *)store {
```

Comment:
I suggest to create `- (void)_registerStore:(TEBaseStore *)store` internal method that will register store in midlewares and dispatcher
@sergeyzenchenko